packages/block-editor/src/components/use-on-block-drop/index.js

```diff
@@ -336,14 +336,40 @@ export default function useOnBlockDrop(
 
 	const moveBlocks = useCallback(
 		( sourceClientIds, sourceRootClientId, insertIndex ) => {
+			const sourceBlocks = getBlocksByClientId( sourceClientIds );
```

Comment:
I can't seem to get this code to run at all. Trying to drop between blocks is now causing an `Uncaught TypeError: Cannot convert undefined or null to object` that freezes the whole editor 😬

Reply:
This should be fixed now.